pages/common/gpg2.md

```diff
@@ -0,0 +1,33 @@
+# gpg
+
+> GNU Privacy Guard 2.
+> Homepage: <https://docs.releng.linuxfoundation.org/en/latest/gpg.html>.
+
+- List imported keys:
+
+`gpg2 --list-keys`
+
+- Encrypt doc.txt for alice@example.com (output to doc.txt.gpg):
+
+`gpg2 --encrypt --recipient {{alice@example.com}} {{doc.txt}}`
+
+- Encrypt doc.txt with only a passphrase (output to doc.txt.gpg):
+
+`gpg2 --symmetric {{doc.txt}}`
+
+- Decrypt doc.txt.gpg (output to STDOUT):
```

Comment:
Perhaps this will read better?
```suggestion
- Decrypt a file, writing the result to the standard output:
```

Reply:
Ditto.

Reply:
I'd change it to: `Decrypt a specified file, writing the result to the standard output` to maintain consistency with the changes proposed bellow.

What do you think?

Reply:
Sounds great!